docs/Users_Guide/index.rst

```diff
@@ -18,10 +18,14 @@ It is important to note here that the Unified Workflow is an evolving software
 package. This documentation describes the |release|
 release dated |release_date|.
 
+.. COMMENT: Since this is for the develop branch, should it be tied to a release...? 
+   I updated the release to 0.1.0/2022-05-06, since I saw on the repo that that's the latest release.
+   Feel free to correct me if that's wrong. 
+
```

Comment:
If memory serves, I believe we are planning to deploy or release from 'develop'. @fgabelmannjr is that consistent with your understanding and ideas?

Reply:
@christinaholtNOAA I think it says that in the documentation, but wouldn't it make sense to make release documentation off of the release *hash* (which can come from develop) but have "latest" docs point to the head of develop? Otherwise people might be using head of develop but release docs or vice versa.

Reply:
Yes! That makes sense. I didn't catch the nuance of the hash coming from develop.